django/test/utils.py

```diff
@@ -836,6 +836,9 @@ def disable(self):
 def tag(*tags):
     """Decorator to add tags to a test class or method."""
     def decorator(obj):
-        setattr(obj, 'tags', set(tags))
+        if hasattr(obj, 'tags'):
+            obj.tags = obj.tags.union(tags)
```

Comment:
This has been mentioned previously, we can't do it because updating the set in place would update the inherited tags as well.

e.g.

```python
@tag('foo')
class Foo:
    pass

@tag('bar')
class Bar(Foo)
    pass
```

Using `update` would add `'bar'` to `Foo.tags`.